addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -966,20 +965,25 @@ registry.anchorSlide = publicWidget.Widget.extend({
      * @private
      */
     _onAnimateClick: function (ev) {
-        if (this.$target[0].pathname !== window.location.pathname) {
-            return;
-        }
         var hash = this.$target[0].hash;
-        if (!utils.isValidAnchor(hash)) {
+        if (!hash.length || (this.$target[0].pathname !== window.location.pathname)) {
             return;
         }
+        // Escape special characters to make the jQuery selector to work.
+        hash = '#' + $.escapeSelector(hash.substring(1));
         var $anchor = $(hash);
```

Comment:
```suggestion
        var $anchor = $(document.getElementById(hash.substring(1)));
```
Don't apply escapes and the like when you can just use "safe" methods to start with.